src/util/index.ts

```diff
@@ -100,7 +100,10 @@ export const getExtensionSigner = async (address: string, extension: Extensions)
   const accounts = await wallet.getAccounts();
 
   if (!accounts) {
-    throw new AppError({ key: 'polkadotjs.noAccounts', payload: { extension } });
+    throw new AppError(
+      { key: 'polkadotjs.noAccounts', payload: { extension } },
```

Comment:
`extension` name is already passed to translation:
`throw new AppError({ key: 'polkadotjs.noAccounts', payload: { extension } });`